### PR TITLE
Buf stats

### PR DESCRIPTION
**Pull request by tedmoore**

### CODE REVIEW

**Review by tremblap — CHANGES_REQUESTED** [review]
good work. I'm missing the graph we had though, but I reckon you will make a clear link to the learn page when it is ready? Where/how will that be?

(same with the info removed on IQR I reckon)

typo: it's corresponding channel -> its

**Comment by tedmoore** [comment]
> I'm missing the graph we had though, but I reckon you will 

What graph?

> make a clear link to the learn page when it is ready? Where/how will that be?

Yes, I think for this object because it is quite big, a link to the Learn will be appropriate. @jamesb93 said he's planning to automatically add a link to every RST. Is this still true?

**Comment by tremblap** [comment]
> What graph?

the table/graph that was there in the original help showing the data structure

> Is this still true?

even if it is, in certain case where it is abstract like this, we'd like to point at it directly I think

**Comment by tedmoore** [comment]
@tremblap, I put the table back in!

**Comment by jamesb93** [comment]
> > I'm missing the graph we had though, but I reckon you will
> 
> What graph?
> 
> > make a clear link to the learn page when it is ready? Where/how will that be?
> 
> Yes, I think for this object because it is quite big, a link to the Learn will be appropriate. @jamesb93 said he's planning to automatically add a link to every RST. Is this still true?

Yeah it's on my list of things to think about it. I'm torn over two ways of doing it which are in the template or just manually...

File: doc/BufStats.rst [hunk 1]
```diff
@@ -1,71 +1,79 @@
-:digest: Computing Statistics on Buffers as Series.
+:digest: Statistically summarise a time series in a Buffer
```
Comment (jamesb93) [inline doc/BufStats.rst:1]: I like

**Review by jamesb93 — APPROVED** [review]

**Pull request merged**